nodes/clarify_util.js

```diff
@@ -95,13 +97,17 @@ module.exports = {
     validateString(validationErrors, 'type', type);
     validateString(validationErrors, 'description', description);
     validateMapStringWithStrings(validationErrors, 'labels', labels);
-    validateMapStringWithStrings(validationErrors, 'annotations', annotations);
+    validateMapStringWithString(validationErrors, 'annotations', annotations);
     validateMapIntWithString(validationErrors, 'enumValues', enumValues);
     validateString(validationErrors, 'engUnit', engUnit);
     validateString(validationErrors, 'sourceType', sourceType);
     validateString(validationErrors, 'sampleInterval', sampleInterval);
     validateString(validationErrors, 'gapDetection', gapDetection);
 
+    if (enumValues && type != 'enum') {
+      validationErrors.push('enumValues can only be set for enums');
+    }
```

Comment:
This isn't true anymore. You can supply `"enumValues"` for a numeric type. It is used if you choose to expose that type as an enum item.

likewise, you can set `"engUnit"` for a an enum, and it will be used if the item is exposed as a number.